README.md

```diff
@@ -26,3 +26,14 @@ The GitHub AI Bot – Coding Assistant is an AI-powered code review bot for all
 ✅ Java Code Analysis – Specialized for all programming language projects.   
 
 ✅ Customizable AI Model – Uses DeepSeek or any other AI model for analysis.
+
+## Spring Boot Application
```

Comment:
Please name this section "Java Bot" and write that this is a Spring Boot Application